src/plugins/vis_type_vislib/public/fixtures/mock_data/filters/_columns.js

```diff
@@ -35,7 +35,7 @@ import _ from 'lodash';
 export default {
   columns: [
     {
-      label: 'Mozilla/5.0 (X11; Linux x86_64; rv:6.0a1) Gecko/20110421 Firefox/6.0a1: agent.raw',
+      label: 'Mozilla/5.0 (X11; Linux x64; rv:6.0a1) Gecko/20110421 Firefox/6.0a1: agent.raw',
```

Comment:
This may not be required, this looks like User Agent, did we run all the UT and Integ tests?

Reply:
Yes. Just run UT and Integ tests